public/humans.txt

```diff
@@ -8,17 +8,25 @@ Oak National Academy’s tech platform has been created by a team of humans supp
   LinkedIn: https://www.linkedin.com/in/aakeshpattani/
   From: London, UK
 
+  Abisola Kuteyi (Data analyst)
+  LinkedIn: https://www.linkedin.com/in/abisolakuteyi/
+  From: London, UK
+
   Amy Matthews (Infrastructure Engineer)
   From: Kent, UK
 
   Andrew Woods (Software Engineering Manager)
-  Site: https://andrewwoods.super.site/
+  Site: https://andrewwoods.info
   LinkedIn: https://www.linkedin.com/in/andykwoods/
   From: Dublin, Ireland
 
   Ben Protheroe (Software Engineer)
   From: London, lives in Cornwall
 
+  Carl Whittaker (Software Engineer)
+  LinkedIn: https://www.linkedin.com/in/carlmw 
+  From: London, UK
+  
```

Comment:
@andykwoods I think I definitely belong in the thanks section as a contractor, otherwise I should probably be removed.